internal/io/event.go

```diff
@@ -1,28 +1,26 @@
 //
-// Copyright (C) 2021 IOTech Ltd
+// Copyright (C) 2021-2022 IOTech Ltd
 //
 // SPDX-License-Identifier: Apache-2.0
 
 package io
```

Comment:
rename and move this file to internal/pkg/utils/payload.go

Reply:
renamed and moved